recipes-core/packagegroups/packagegroup-ni-extra.bb

```diff
@@ -590,7 +590,6 @@ RDEPENDS_${PN} += "\
 RDEPENDS_${PN} += "\
 	python3-future \
 	python3-gsocketpool \
-	python3-mprpc \
```

Comment:
Is the reference to `myrpc` in the commit message supposed to be `mprpc`?

Reply:
It is; good catch. Will fix.